File: bindings/dart/acdc/ios/Classes/AcdcPlugin.m
#import "AdccPlugin.h"
#if __has_include(<acdc/acdc-Swift.h>)
#import <acdc/acdc-Swift.h>
#else
#import "acdc-Swift.h"
#endif

@implementation AcdcPlugin
+ (void)registerWithRegistrar:(NSOBject<FlutterPluginRegistrar>*)registrar {
    [SwiftAcdcPlugin registerWithRegistrar:registrar];
}
@end

File: bindings/dart/acdc/ios/Classes/AcdcPlugin.swift
import Flutter
import UIKit

public class AcdcPlugin: NSObject, FlutterPlugin {
  public static func register(with registrar: FlutterPluginRegistrar) {
    let channel = FlutterMethodChannel(name: "acdc", binaryMessenger: registrar.messenger())
    let instance = AcdcPlugin()
    registrar.addMethodCallDelegate(instance, channel: channel)
  }

  public func handle(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
    switch call.method {
    case "getPlatformVersion":
      result("iOS " + UIDevice.current.systemVersion)
    default:
      result(FlutterMethodNotImplemented)
    }
  }

  public func dummyMethodToEnforceBundling() {
    dummy_method_to_enforce_bundling();
  }
}
